direct/nn/openvino/openvino_model.py

```diff
@@ -11,37 +11,58 @@ def __init__(self, model):
         super().__init__()
         self.model = model
         self.exec_net = None
+        self.input = None
+        self.model_name = self.model.__class__.__name__
 
-    def create_net(self, input_image, masked_kspace, sampling_mask, sensitivity_map):
+    def create_net(self, **kwargs):
         ie = IECore()
         ie.add_extension(get_extensions_path(), "CPU")
 
-        buf = io.BytesIO()
-        torch.onnx.export(
-            self.model,
-            (input_image, masked_kspace, sampling_mask, sensitivity_map),
-            buf,
-            opset_version=11,
-            operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
-            input_names=["input_image", "masked_kspace", "sampling_mask", "sensitivity_map"],
-            output_names=["cell_outputs", "previous_state"],
-        )
-
-        net = ie.read_network(buf.getvalue(), b"", init_from_buffer=True)
-        self.exec_net = ie.load_network(net, "CPU")
-
-    def forward(self, input_image, masked_kspace, sampling_mask, sensitivity_map):
-        input_map = {
-            "input_image": input_image,
-            "masked_kspace": masked_kspace,
-            "sampling_mask": sampling_mask,
-            "sensitivity_map": sensitivity_map,
-        }
+        if self.model_name == "RIM":
+            args = ["input_image", "masked_kspace", "sampling_mask", "sensitivity_map"]
+            output_names = ["cell_outputs", "previous_state"]
+        elif self.model_name == "Unet2d":
+            output_names = ["output"]
+            if self.model.image_initialization == "sense":
+                args = ["masked_kspace", "sensitivity_map"]
+            elif self.model.image_initialization == "zero_filled":
+                args = ["masked_kspace"]
```

Comment:
There are only two possible inputs in UNet2d so let's do this:
```python
args = ["masked_kspace"]
if self.model.image_initialization == "sense":
    args.append("sensitivity_map")
```
No need for exception.